apps/ios/Shared/Views/Call/ActiveCallView.swift

```diff
@@ -106,6 +118,12 @@ struct ActiveCallView: View {
                         }
                     }
                 }
+                if state.connectionState == "closed" && m.activeCall != nil {
+                    DispatchQueue.main.async {
```

Comment:
await MainActor.run?

Reply:
also, on line 112 (call.callState = .connected):
/Users/evgeny/code/simplex-chat/simplex-chat/apps/ios/Shared/Views/Call/ActiveCallView.swift:112: warning run: Publishing changes from background threads is not allowed; make sure to publish values from the main thread (via operators like receive(on:)) on model updates.